src/ui/atoms/icon/Icon.tsx

```diff
@@ -18,8 +18,12 @@ export type IconName =
   | 'clock'
   | 'close'
   | 'cross'
-  | 'facebook'
-  | 'linkedin'
+  | 'discord-round'
+  | 'facebook-card'
```

Comment:
I will have named it `facebook-square` to follow the `round` convention. 
Card does not mean a lot to me.